index.js

```diff
@@ -379,3 +383,10 @@ Measure.propTypes = {
   delay: PropTypes.number,
   unload: PropTypes.bool
 };
+
+Measure.webVitals = {
```

Comment:
Needs some JSDoc, and some actual documentation in the README to describe what people are supposed to do with this because right now, it's doing nothing.